core/src/avm1/object.rs

```diff
@@ -696,8 +696,10 @@ pub fn search_prototype<'gc>(
     activation: &mut Activation<'_, 'gc>,
     this: Object<'gc>,
     is_slash_path: bool,
+    include_resolve: bool,
```

Comment:
`include_resolve` is always true now, right?

Reply:
You're right!